auth/base/dex-cm.yaml

```diff
@@ -0,0 +1,61 @@
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: dex
+data:
+  config.yaml: |
+    issuer: <DEX_HOST>/api/dex
+
+    storage:
+      type: memory
+
+    web:
+      http: "0.0.0.0:5556"
+
+    grpc:
+      addr: "0.0.0.0:5557"
+
+    telemetry:
+      http: "0.0.0.0:5558"
+
+    oauth2:
+      skipApprovalScreen: true
+
+    staticClients:
+      - id: argo
+        name: Argo
+        redirectURIs:
+          - http://URL/oauth2/callback
+        secretEnv: ARGO_SECRET
+
+      - id: grafana
+        name: Grafana
+        redirectURIs:
+          - https://URL/login/generic_oauth
+        secretEnv: GRAFANA_SECRET
+
+      - id: grafana_observatorium
```

Comment:
@tumido can we remove observatorium grafana from this? It already has login disabled and we don't need it to be enabled.

Reply:
I'm cool with that, np, I'll remove it